packages/now-next/src/utils.ts

```diff
@@ -520,10 +540,31 @@ export async function createLambdaFromPseudoLayers({
   const zipFile = new ZipFile();
   const addedFiles = new Set();
 
+  const names = Object.keys(files).sort();
```

Comment:
What's the purpose of `sort()` here?

Reply:
Not sure, for consistency? I pulled this logic from `createLambda` [here](https://github.com/zeit/now/blob/17687e9bcd9fa32e271849505fd666093d1788a8/packages/now-build-utils/src/lambda.ts#L107)